keel-core/src/main/kotlin/com/netflix/spinnaker/keel/IntentRepository.kt

```diff
@@ -17,11 +17,13 @@ package com.netflix.spinnaker.keel
 
 interface IntentRepository {
 
-  fun upsertIntent(intent: Intent<IntentSpec>)
+  fun createIntent(intent: Intent<IntentSpec>): Intent<IntentSpec>
```

Comment:
Do we need `createIntent` still if `upsertIntent` is a thing?

Reply:
Good point. Removing.